mod/consensus/pkg/cometbft/cli/commands.go

```diff
@@ -143,34 +145,42 @@ func ShowValidatorCmd() *cobra.Command {
 		Short: "Show this node's CometBFT validator info",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cfg := clicontext.GetConfigFromCmd(cmd)
-			privValidator := pvm.LoadFilePV(
+
+			pk := pvm.LoadFilePV(
 				cfg.PrivValidatorKeyFile(),
 				cfg.PrivValidatorStateFile(),
-			)
-			pk, err := privValidator.GetPubKey()
-			if err != nil {
-				return err
-			}
-
-			sdkPK, err := cryptocodec.FromCmtPubKeyInterface(pk)
-			if err != nil {
-				return err
-			}
+			).Key.PubKey
```

Comment:
Overall looks good for the fix. I was curious, do we see any benefit in doing this ? I think it wasn't necessary.